index.bs

```diff
@@ -1477,102 +1514,122 @@ RemoteValue = {
   WindowProxyRemoteValue //
 }
 
+InternalId = Number;
```

Comment:
Capital-N Number isn't a type. Also I'd strongly prefer that we use an integer here; allowing the identifier to be a float is just asking for problems.

Reply:
My bad. Done